tests/test_processor.py

```diff
@@ -0,0 +1,159 @@
+"""Test the processor module"""
+import pandas as pd
+import pytest
+from pytest import approx
+
+from orangecontrib.argument.miner.processor import (
+    check_columns,
+    get_argument_topics,
+    get_argument_sentiment,
+    get_argument_coherence,
+    update_argument_table,
+    _match_list_size,
+    _aggregate_list_by_another,
+)
+from .conftest import (
+    mock_processor__aggregate_list_by_another,
+    mock_processor__match_list_size,
+)
+
+
+def test_check_columns():
+    """Unit test check_columns."""
+    dummy_df = pd.DataFrame({"col1": [1, 2], "col2": [3, 4]})
+    dummy_expected_cols = ["col1", "col3"]
+    with pytest.raises(AssertionError) as ex:
+        check_columns(expected_cols=dummy_expected_cols, data=dummy_df)
+    assert str(ex.value) == "Missing columns in the input dataframe: ['col3']."
+
+
+def test__match_list_size():
+    """Unit test _match_list_size"""
+    dummy_list1 = [1, 2, 3]
+    dummy_list2 = [2, 3, 4]
+
+    _match_list_size(dummy_list1, dummy_list2)
+
+
+def test__match_list_size_negative():
```

Comment:
Sure, I wrote the tests that way just because it's easier for me to know which tests fails. But you're right, it's nicer to have the dummy data passed as parameters.